.weave/migrations/versions/002_initial_slack_tables.py

```
@@ -0,0 +1,68 @@
+"""Initial Slack tables
+
+Revision ID: 002
+Revises: 000
+Create Date: 2024-01-20 12:00:00.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
```

Ruff (0.11.9)

`sqlalchemy.dialects.postgresql` imported but unused

Remove unused import: `sqlalchemy.dialects.postgresql`

(F401)